trident.md

```diff
@@ -509,6 +509,15 @@ The specific semantics of the various ways to include or exclude entities are:
 
 If a query results in multiple individuals with the same name, forge will throw an error.
 
+##### Treatment of the genotype data while merging
+
+Forge performs a series of steps to merge the genotype data of multiple source files:
+
+1. Genotype data from each package is streamed in parallel and checks are performed that any SNP location present in any file is outputted (with missing data being filled in for packages where a particular SNP is missing), unless `--intersect` is given, in which case only those position which are present in every package are output.
```

Comment:
I find this a bit confusing now. Does `--intersect` apply on a package or a SNP-level? if it applies on a SNP-level then our change to the `snpSet` field in the POSEIDON.yml file (documented somewhere else in this guide) is incorrect, right?

| Input snpSet A | Input snpSet B | `--intersect` | Ouput snpSet |
|----------------|----------------|---------------|--------------|
| Other          | *              | *             | Other        |
| 1240K          | HumanOrigins   | True          | HumanOrigins |
| 1240K          | HumanOrigins   | False         | 1240K        |

Because there could be random SNPs missing from all individuals in the forge operation involving e.g. HumanOrigins packages, making the output only feature a subset of the HumanOrigins SNP set.

Reply:
Sorry, OK, I see where the confusion comes from. "Missing" here means that the SNP isn't even listed at all in the respective package. So the documentation is correct. I will clarify the text.